src/pypgstac/src/pypgstac/pypgstac.py

```diff
@@ -147,6 +151,59 @@ def load_queryables(
             queryables_data = item
             break  # We only need the first item
 
+            # Create missing collections if requested
```

Comment:
```suggestion
        # Create missing collections if requested
```

Reply:
@emmanuelmathot I think it makes sense to have a generic SQL create_empty_collections function. Right now, you've got some slight SQL injection vulnerabilities in there (f strings and SQL are not good friends). Do you mind if I push some changes to this PR?